### ISSUE TITLE
`v` (view) when viewing list prints in background?

### ISSUE THREAD

**Issue by mankoff** [ankitpokhrel/jira-cli#455]
After running `jira issue list` I am presented with a list of issue. If I type `v` to view one, a box flashes that says `processing... please wait` and then I'm returned to the list. Later, when I hit `q` I see the issue I tried to view printed to the terminal. Is this the expected behavior? 

**Please provide following details**
1. JiraCLI Version:

```bash
$ jira version
(Version="1.1.0", GitCommit="3b93e147eac468ad985bdece27469153b4bb2814", CommitDate="2022-08-14T08:10:40+00:00", GoVersion="go1.18.3", Compiler="gc", Platform="linux/amd64")
```

2. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.

```
On-premise
```

3. What operating system are you using? Also mention version.
 
```
Linux
```

4. What terminal are you using? Also mention version.

+ Konsole (KDE)
+ xterm

**Comment by ankitpokhrel** [comment]
Hi @mankoff, no it is not the expected behavior. Could you please check if you've `PAGER` env set and what its value? 

The app uses [less -r](https://www.man7.org/linux/man-pages/man1/less.1.html) command in view mode. Could you also check if you open any file with `less` or `less -r` does it have the same behavior?

**Comment by mankoff** [comment]
I had `PAGER` set to `less` not `less -v`. This fixes it. Thank you.

**Comment by mankoff** [comment]
Re-opening and suggesting a hold on #457.

When I `export PAGER='less -v'`, or alternatively leave `PAGER=less` and then `export LESS=-F -X -R -v`, then the `v` key works in `jira-cli`.

However, when I try to use less in `bash` (or read a `man` page), I get now get the following error:

```
$ man less
There is no -v option ("less --help" for help)
Press RETURN to continue 
```

I'm using less version:

```
less 590 (GNU regular expressions)
Copyright (C) 1984-2021  Mark Nudelman
```

**Comment by ankitpokhrel** [comment]
Hi @mankoff, I am not aware of `-v` option in `less`. The tool internally uses `less -r` option if `PAGER` is not set. Did you try with `less -r`?

**Comment by mankoff** [comment]
I'm not sure why I saw `-v` and tried that. Weird bug in my brain. I found the bug. Because I had export `LESS=-F -X -R`, the `-F` flag quits less if the pager is less than one terminal screen. That was the problem. The `-r` flag is not needed, just no `-F`.

**Comment by realtime-neil** [comment]
> USE OF THE -r OPTION IS NOT RECOMMENDED.

 -- https://www.freebsd.org/cgi/man.cgi?query=less

> The app uses [less -r](https://www.man7.org/linux/man-pages/man1/less.1.html) command in view mode.

 -- https://github.com/ankitpokhrel/jira-cli/issues/455#issuecomment-1235124085

@ankitpokhrel Why?

**Comment by ankitpokhrel** [comment]
Hmm...I don't really remember why but I don't see that warning in my version of less (I still have old version 487). 

![Screen Shot 2022-09-05 at 6 55 24 PM](https://user-images.githubusercontent.com/2364546/188492169-6551ef59-7b0c-4a70-b364-321d0d2862f9.png)

I think we can use the option `R`, I realized that I have my pager set to `less -R` locally.

**Comment by samundra** [comment]
My less version also has warning on its doc.

```bash
ᐅ less --version
less 581.2 (POSIX regular expressions)
Copyright (C) 1984-2021  Mark Nudelman
```

See screenshot below taken from `man less` 

|Screenshot|
|---|
|![Screen Shot 2022-09-06 at 17 43 12](https://user-images.githubusercontent.com/760855/188615774-0db9a8e5-cb61-410b-b203-2fe58cf8d054.jpg)|

**Comment by Gidgidonihah** [comment]
I got here looking for the same problem. Turns out yep, my less configuration is to exit if one screen or less (`-F`). I was configured like so:
```
export LESS="-SFXR -x2"
export PAGER=less
```
But I generally wan to keep the `-F` option. Thankfully there is the `JIRA_PAGER` option. Unfortunately, I've configured less using the `LESS` env var. I could drop that var, but a quick change to this works:
```
export LESS="-SXR -x2"
export PAGER="less -F"
export JIRA_PAGER=less
```

This seems like a non-obvious gotcha, and may be worth a note on the docs.
